core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -738,8 +740,10 @@ private void handleLocaleCheck() {
     unbinder.unbind();
     // TODO create a base Activity class that class this.
     FileUtils.deleteCachedFiles(getActivity());
-    tts.shutdown();
-    tts = null;
+    if (tts != null) {
```

Comment:
we can just use `tts?.shutdown()` instead of the `if`

Reply:
@macgills It's in java.

Reply:
d'oh, stupid java